.github/workflows/publish.yml

```diff
@@ -31,6 +31,27 @@ jobs:
           regex: |
             applications\/(?<rdnn>[a-zA-Z0-9\.\-]+)\.json
 
+      - name: Check for YML manifest
```

Comment:
We don't clone the remote repo here, so it's checking for the manifest file in the appcenter-reviews repo

Reply:
derp my bad

Reply:
It's all good. I'll write a small task to get the file or fail

Reply:
I took another crack at it. It doesn't seem to work in my fork, but I'm not sure why :dizzy_face: